transport-native-unix-common-tests/src/main/java/io/netty/channel/unix/tests/SocketTest.java

```diff
@@ -13,29 +13,30 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-package io.netty.channel.unix;
+package io.netty.channel.unix.tests;
 
-import io.netty.channel.epoll.Epoll;
-import java.io.File;
+import io.netty.channel.unix.DomainSocketAddress;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.netty.channel.unix.DomainSocketAddress'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)